webapp/src/Form/Type/ContestType.php

```diff
@@ -39,28 +41,34 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'Yes' => true,
                 'No' => false,
             ],
+            'help' => 'Only John knows',
         ]);
         $builder->add('freezetimeString', TextType::class, [
             'label' => 'Scoreboard freeze time',
             'required' => false,
+            'help' => 'Time when the freeze starts: the results of submissions made after this time are not revealed until the scoreboard unfreeze time below has passed.',
         ]);
         $builder->add('endtimeString', TextType::class, [
             'label' => 'End time',
+            'help' => 'Time when the contest ends.',
```

Comment:
+ "(Submissions will be accepted but shown as 'too-late' and not counted towards the score)"